fix: FIT-1045: Add project state filter + sort option behind 710 flag

yyassi-heartex opened the pull request:
This pull request introduces a new feature flag for FSM State Fields and enhances filter handling in the Filters component by supporting disabled filter fields. The changes are primarily focused on feature flag management and UI improvements for filters.

Feature flag management:

* Added a new feature flag `FF_FSM_STATE_FIELDS` for FSM State Fields in `flags.ts`.

UI improvements for filters:

* Updated the Filters component to include a `disabled` property on filter fields, allowing filters to be marked as disabled in the UI.

yyassi-heartex committed:
fix: FIT-1045: Add project state filter + sort option behind 710 flag

bmartel committed:
reverting Filters changes to develop

bmartel committed:
Merge remote-tracking branch 'origin/develop' into fb-fit-1045/ff-wrap

bmartel commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19653885431)

bmartel committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19653885431

bmartel committed:
Merge branch 'develop' into 'fb-fit-1045/ff-wrap'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19653885431
